ui/client/components/HealthCheck.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import {connect} from "react-redux"
+import ActionsUtils, {EspActionsProps} from "../actions/ActionsUtils"
+import InlinedSvgs from "../assets/icons/InlinedSvgs"
+import HttpService from "../http/HttpService"
+import PeriodicallyReloadingComponent from "./PeriodicallyReloadingComponent"
+import {withTranslation} from "react-i18next"
+import {WithTranslation} from "react-i18next/src"
+import {compose} from "redux"
+
+type HealthCheckResponse = {
+  state: string,
+  error: string,
+}
+
+type OwnProps = {}
+
+type State = {
+  healthCheck?: HealthCheckResponse,
+}
+
+class HealthCheck extends PeriodicallyReloadingComponent<Props, State> {
+
+  public static STATE_OK = "ok"
+
+  onMount(): void {
```

Comment:
Can  it be default implementation in abstract class?

Reply:
You'r right. It works - changed.